docs/guide/copyright.rst

```diff
@@ -1,9 +1,4 @@
-
-.. _../pages/guide/copyright#copyright:
-
 Copyright
 *********
 
-The localisation guide is released under the Creative Commons
-`Attribution-ShareAlike <http://creativecommons.org/licenses/by-sa/3.0/>`_
-license.
+The localisation guide is released under the `Creative Commons Attribution-ShareAlike <http://creativecommons.org/licenses/by-sa/3.0/>`_ license.
```

Comment:
I suspect that the previous link text, and layout, was chosen intentionally.
In any case, your line exceeds the implicit line width that you can see exists in this file.